addon/addon/builder/fixture-builder-factory.js

```diff
@@ -5,17 +5,15 @@ import JSONAPIFixtureBuilder from './jsonapi-fixture-builder';
 import RESTFixtureBuilder from './rest-fixture-builder';
 import JSONFixtureBuilder from './json-fixture-builder';
 import ActiveModelFixtureBuilder from './active-model-fixture-builder';
-import {
-  macroCondition,
-  dependencySatisfies,
-  importSync,
-} from '@embroider/macros';
+import { importSync } from '@embroider/macros';
 
 let ActiveModelSerializer;
-if (macroCondition(dependencySatisfies('active-model-adapter', '*'))) {
+try {
   ActiveModelSerializer = importSync(
     'active-model-adapter'
   ).ActiveModelSerializer;
+} catch (e) {
+  // no worries, go on without it - it's optional
```

Comment:
Note this change. `dependencySatisfies()` doesn't work for optional peerdeps (will only work if the dep is listed as dependency, which we don't want) - rather, we can just use try/catch. If they don't have the package, that's OK.

Made the same change for the ember-data-model-fragment imports.

Some references
https://github.com/embroider-build/embroider/pull/1187/files#r855292577
https://github.com/embroider-build/embroider/issues/1266